content/en/security/cloud_siem/log_detection_rules.md

```diff
@@ -178,6 +180,25 @@ In the **This rule will not generate a signal if there is a match** field, you h
 [1]: /logs/search_syntax/
 [2]: /logs/log_configuration/processors#geoip-parser
 {{% /tab %}}
+
+{{% tab "Third Party" %}}
+
+### Root query
+
+Construct a search query using the same logic as a [log explorer search][1]. The trigger defined for each new attribute generates a signal for each new value of that attribute over a 24-hour roll-up period.
+
+Click **Add Query** to add additional queries.
+
+**Note**: The query applies to all ingested logs.
```

Comment:
@nimishaaa I simplified this sentence that we were discussing about in the doc. Let me know if this works, thanks! 

Only logs are scanned, right? Not events? I don't see events mentioned anywhere else in the Cloud SIEM docs. I can also update the same sentence in Thresholds.

Reply:
DM'd and Nimisha is good with this.